### ISSUE TITLE
Unsupported background colour for YAML parser

### ISSUE THREAD

**Issue by Amudtogal** [plantuml/plantuml#503]
Dear plantuml developers,

thanks for this awesome tool, it really makes my code shine and helps our group putting things into perspective.

When I tried to use the yaml parser with a simple yaml file like this:

```
@startyaml
skinparam backgroundColor Silver
fruit: Apple
size: Small
color: Red
@endyaml
```

plantuml fails with an exception:
```
java.lang.UnsupportedOperationException: skinparam backgroundColor Silver
        at net.sourceforge.plantuml.yaml.SimpleYamlParser.parseSingleLine(SimpleYamlParser.java:182)
        at net.sourceforge.plantuml.yaml.SimpleYamlParser.parse(SimpleYamlParser.java:68)
        at net.sourceforge.plantuml.yaml.YamlDiagramFactory.createSystem(YamlDiagramFactory.java:70)
        at net.sourceforge.plantuml.PSystemBuilder.createPSystem(PSystemBuilder.java:134)
        at net.sourceforge.plantuml.BlockUml.getDiagram(BlockUml.java:163)
        at net.sourceforge.plantuml.SourceFileReaderAbstract.getGeneratedImages(SourceFileReaderAbstract.java:149)
        at net.sourceforge.plantuml.Run.manageFileInternal(Run.java:538)
        at net.sourceforge.plantuml.Run.processArgs(Run.java:422)
        at net.sourceforge.plantuml.Run.manageAllFiles(Run.java:389)
        at net.sourceforge.plantuml.Run.main(Run.java:198)
```

The same error applies when reading in a configuration file that contains this setting (`skinparam backgroundColor ...`).
Is there an alternative way of letting plantuml ignore the unsupported configuration settings?
Or do we need to avoid using the configuration file altogether (maybe this could be put into the CLI parser)?

Regards,
Simon

**Comment by arnaudroques** [comment]
This has been implemented in release 1.2021.3
But you have [to use styles](http://www.plantuml.com/plantuml/umla/VOvH2i8m343_SuhSmFynPA8dS2U8RHp5j1rfgrJPtRLbtxwFm2CyaYbbqbp92umfL-4ZDBauMfGyVW1_iB3daBirRcGUYyOIx3bALAmi4bzzkVLfeob_Yzd9atKlDXWF--sx5fS7FAshC2JtvW5dJo9WMZRWbIrC76pxyGi0).

Does it help ?

**Comment by Amudtogal** [comment]
Ah, yea!

This works! Now I will just need to figure out how to organise the style
files...

Thanks!

Closes #503
